src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -1241,7 +1241,9 @@ internal HttpCreationPayload GetInstanceCreationLinks()
 
         internal Uri GetWebhookUri()
         {
-            return this.webhookUrlProvider?.Invoke() ?? throw new InvalidOperationException("Webhooks are not configured");
+            string errorMessage = "Webhooks are not configured. This may occur if the environment variable `WEBSITE_HOSTNAME` is not set (should be automatically set for Azure Functions). " +

```

Comment:
nit on `<ip-address>:<port>` - should we instead use something around "URI" wording? So it is more clear it is open to ip-address:port or a DNS name

Reply:
Good idea - implemented in commit: https://github.com/Azure/azure-functions-durable-extension/pull/2849/commits/3621b121c9a106ceae985ab35dc3f96a631853c9